crates/egui_plot/src/legend.rs

```diff
@@ -267,14 +267,43 @@ impl Widget for &mut LegendWidget {
                 .multiply_with_opacity(config.background_alpha);
                 background_frame
                     .show(ui, |ui| {
-                        entries
+                        let mut focus_on_item = None;
+
+                        let response_union = entries
                             .iter_mut()
-                            .map(|(name, entry)| entry.ui(ui, name.clone(), &config.text_style))
+                            .map(|(name, entry)| {
+                                let response = entry.ui(ui, name.clone(), &config.text_style);
+
+                                if response.clicked() && ui.input(|r| r.modifiers.alt) {
```

Comment:
Where is the normal clicking handled? We shouldn't spread out the different click-handling to different places

Reply:
good call. i'll refactor some stuff.